src/scrape_up/github/respository.py

```diff
@@ -351,6 +354,22 @@ def get_contributors(self):
                 "message": message,
             }
 
+
+    def last_update_at(self):
+        data=self.__scrape_page()
+        try:
+            update=data.find_all("relative-time", class_="no-wrap")
+            return update[0].get_text()
```

Comment:
Follow the return type format in try block also.

Reply:
Done.